snuba/migrations/snuba_migrations/sessions/0002_sessions_aggregates.py

```diff
@@ -0,0 +1,279 @@
+from importlib import import_module
+from typing import Sequence
+
+from snuba.clickhouse.columns import (
+    AggregateFunction,
+    Column,
+    DateTime,
+    String,
+    UInt,
+    UUID,
+)
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations
+from snuba.migrations.columns import LowCardinality, WithDefault
+from snuba.processor import MAX_UINT32, NIL_UUID
+
+
+v1_matview = import_module(
+    "snuba.migrations.snuba_migrations.sessions.0001_sessions"
+).create_matview
+
+
+DEFAULT_QUANTITY = 1
+
+
+aggregate_columns = [
+    Column("org_id", UInt(64)),
+    Column("project_id", UInt(64)),
+    Column("started", DateTime()),
+    Column("release", LowCardinality(String())),
+    Column("environment", LowCardinality(String())),
+    Column(
+        "duration_quantiles",
+        AggregateFunction("quantilesIf(0.5, 0.75, 0.9, 0.95, 0.99)", UInt(32), UInt(8)),
+    ),
+    Column("duration_avg", AggregateFunction("avgIf", UInt(32), UInt(8))),
+    # sessions:
+    Column("sessions", AggregateFunction("sumIf", UInt(32), UInt(8))),
+    Column("sessions_crashed", AggregateFunction("sumIf", UInt(32), UInt(8))),
+    Column("sessions_abnormal", AggregateFunction("sumIf", UInt(32), UInt(8))),
+    Column("sessions_errored_sum", AggregateFunction("sumIf", UInt(32), UInt(8))),
+    Column("sessions_errored_uniq", AggregateFunction("uniqIf", UUID(), UInt(8))),
+    # users:
+    Column("users", AggregateFunction("uniqIf", UUID(), UInt(8))),
+    Column("users_crashed", AggregateFunction("uniqIf", UUID(), UInt(8))),
+    Column("users_abnormal", AggregateFunction("uniqIf", UUID(), UInt(8))),
+    Column("users_errored", AggregateFunction("uniqIf", UUID(), UInt(8))),
+]
+
+
+query = f"""
+SELECT
```

Comment:
Did you try to ingest data after the migration? I have never tried to skip non nullable columns to fill in a materialized view statement.